Corgibytes.Freshli.Cli/Functionality/Doctor/StartDoctorActivity.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Corgibytes.Freshli.Cli.Functionality.Engine;
+
+namespace Corgibytes.Freshli.Cli.Functionality.Doctor;
+
+public class StartDoctorActivity : IApplicationActivity
+{
+    public readonly List<Tuple<string, int>> ErrorCode;
+
+    public StartDoctorActivity(string gitPath, string cacheDirectory, List<Tuple<string, int>> errorCode)
+    {
+        GitPath = gitPath;
+        CacheDirectory = cacheDirectory;
+        ErrorCode = errorCode;
+    }
+
+    private string GitPath { get; }
+    private string CacheDirectory { get; }
+
+    public void Handle(IApplicationEventEngine eventClient)
+    {
+        try
+        {
+            if (!Directory.Exists(CacheDirectory))
+            {
+                // It's possible to create the cache directory if it does not exist
+                Directory.CreateDirectory(CacheDirectory);
+                Console.Out.Write("Successfully created the cache directory");
+            }
+
+            File.WriteAllText(CacheDirectory + Path.DirectorySeparatorChar + "TextFile.txt",
```

Comment:
I think we should clean up these files as well.

Reply:
👋 Hi @edwinkortman when you are saying clean up these files, do you mean the whitespace?

Reply:
The activity is creating files, with cleaning up I mean deleting them after everything has been tested.

Reply:
Ah I see, I can get that done, let me finish up the other comments. Thanks for the feedback!